console-store/Clases/Store.cs

```diff
@@ -0,0 +1,107 @@
+﻿
+using System;
+using System.Reflection.Emit;
+using System.Text.Json;
+
+namespace ConsoleStore.Clases
+{
+    internal class Store
+    {
+        private List<Category> _categories = new List<Category>(JsonSerializer.Deserialize<List<Category>>(File.ReadAllText("../../../categories.json")));
```

Comment:
I would recommend to avoid using relative pathes in .NET apps. For education purposes, you can store this file in bin folder, and path will look like `categories.json`